po/it.po

```diff
@@ -224,100 +238,108 @@ msgstr ""
 #. Translators: This feature is disabled by default, its translation has low priority.
 #: src/pt-asr-assistant.c:194
 msgid "The folder contains no acoustic model."
-msgstr ""
+msgstr "La cartella non contiene alcun modello acustico"
 
 #. Translators: This feature is disabled by default, its translation has low priority.
 #: src/pt-asr-assistant.c:301
 msgid "Please select …"
-msgstr ""
+msgstr "Seleziona..."
```

Comment:
Use the same ellipsis character as in the English string, it's a single Unicode character vs. 3 dots/characters